.github/workflows/beam_PostCommit_Python_Arm.yml

```diff
@@ -18,8 +18,6 @@
 name: PostCommit Python Arm
 
 on:
-  issue_comment:
```

Comment:
ARM tests do not have Jenkins counterpart

Reply:
Reverted back